feature/tabsflow/moreflow/about/build.gradle.kts

```diff
@@ -8,6 +8,8 @@ android {
 }
 
 dependencies {
+    implementation(platform(libs.androidx.compose.bom))
```

Comment:
App works the same way without this dependency

Reply:
fixed